addons/resource/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,39 @@
+---Fields in module 'resource'---
+resource     / res.company              / resource_calendar_id (many2one): NEW relation: resource.calendar
+resource     / res.company              / resource_calendar_ids (one2many): NEW relation: resource.calendar
+resource     / res.users                / resource_ids (one2many)       : NEW relation: resource.resource
+resource     / resource.calendar        / global_leave_ids (one2many)   : NEW relation: resource.calendar.leaves
+resource     / resource.calendar        / manager (many2one)            : DEL relation: res.users
+resource     / resource.calendar.leaves / tz (selection)                : NEW selection_keys: function
```

Comment:
For leaves of resources that are linked to a res.users, can you fill this `tz` field with the users' timezones?

Reply:
Done